djangoevents/unifiedtranscoder.py

```diff
@@ -29,10 +29,10 @@ def serialize(self, domain_event):
         assert isinstance(domain_event, DomainEvent)
 
         event_data = domain_event.__dict__.copy()
-        event_data.pop('domain_event_id')
-        event_data.pop('entity_id')
-        event_data.pop('entity_version')
-        event_data.pop('metadata')
+        event_data.pop('domain_event_id', None)
+        event_data.pop('entity_id', None)
+        event_data.pop('entity_version', None)
+        event_data.pop('metadata', None)
```

Comment:
actually, that might be a good moment to change this a little bit and use `get(key)` instead of `pop'ing` keys?